pages/koneet-kiertoon.tsx

```diff
@@ -0,0 +1,411 @@
+  color: rgba(255,255,255,0.6);
+  width: auto !important;
+  text-align: left;
+  margin: 0 0 1rem 0 !important;
+`
+const SpecList = styled.div`
+
+  display: flex;
+  margin-top: 1rem;
+  flex-direction: column;
+  min-width: 50%
+  height: 
+  justify-content: flex-start;
+
+  p {
+    width: 100%;
+    margin-top: 0.2rem;
+  }
+
+  @media(max-width: 1090px) {
+    width: 100% !important;
+  }
+`
+
+const ResponsiveSplitFlex = styled.div`
+
+  display: flex;
+  width: 100% !important;
+  flex-wrap: nowrap !important;
+
+  div > div {
+    height: 80%;
+    width: 100%;
+  }
+
+
+  @media(max-width: 1090px) {
+    flex-direction: column-reverse;
+    align-items: flex-start;
+
+    div:nth-child(2) {
+      flex-wrap: nowrap;
+      width: 100%;
+      height: 20rem;
+
+      ul {
+        margin-top: 1rem;
+        margin-left: 1rem;
+      }
+
+      ul > li {
+        height: 3rem;
+        margin-right: 100%;
+        margin-bottom: 0.5rem;
+      }
+    }
+  }
+
+  @media(max-width: 776px) {
+
+    div:nth-child(2) {
+      width: 100%;
+      height: auto;
+      flex-wrap: wrap;
+      margin-top: unset;
+      height: 25rem !important;
+
+      ul {
+        margin-top: 1rem;
+        margin-bottom: 1rem;
+        margin-left: unset;
+      }
+
+      ul > li {
+        height: 3rem;
+        margin-right: 0.5rem;
+      }
+    }
+  }
+`
+
+export default function ComputerShare({ copyrightYear }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+  return (
+    <div>
+      <Head>
+          <title>Koneet kiertoon | Testausserveri</title>
+          <meta name="description" content="Testausserveri on kaikille avoin yhteisö koodaamisesta, eettisestä hakkeroinnista ja yleisesti teknologiasta innostuneille nuorille." />
+      </Head>
+      <Content>
+
+        <H1>Koneet kiertoon -kampanja</H1>
+        <InfoBox>
+          <span>Tämän sivun tietoja täydennetään vielä.</span>
+          <span>Kampanjan alkamisesta tiedotetaan erikseen.</span>
+        </InfoBox>
+        <p>
+          Testausserveri ry organisoi yksityisen lahjoittajan lahjoittamien tietokoneiden ja palvelimien jakamisen yhdistyksen jäsenille ja hyväntekeväisyyteen.
+          Kampanjan tavoitteena on antaa turhiksi jääneille vanhoille, mutta edelleen moniin eri käyttötarkoituksiin sopiville tietokoneille ja palvelimille uusi elämä.
+          Jaossa on toimistokäytössä olleita pöytäkoneita ja kannettavia, sekä tuotantovalmiita palvelimia.*
+        </p>
+        <Section>
+          <h2>Lahjoitettavat laitteet</h2>
+          <p>
+            Tässä on alustava lista kaikista lahjoitettavista laitteista ja niiden eri konfiguraatioista. Ilmoitetut määrät saattavat muuttua, mikäli osa laitteista on rikkinäisiä tai vaatii varaosia.
+            Kaikki laitteet jaetaan puhdistettuina ja testattuina.
+            <br />
+            <b>Kaikkien laitteiden yhteydessä annetaan kyseiseen laitteeseen sopiva virtalähde/kaapeli.</b>
+          </p>
+          <Device>
```

Comment:
Probably not necessary (and maybe also a personal opinion), but I would personally refactor the 3 computer models into a reusable component. That way we could ensure consistent styling, reduce duplicated code and make the code cleaner overall.

So for example
```tsx
<ComputerModel
    description="Nämä HP:n Workstation koneet ovat..."
    name="HP Workstation Z240"
    amount={{
        remaining: 13,
        max: 15
    }}
    specs={{
        case: "Torni",
        cpu={{
            model: "i7-7700",
            cores: 4
            ...
        }}
        gpu={[
             ...,
             ...,
        ]}
    }}
    note="Huom. kuvassa olevasta koneesta..."
/>
```

But as I said, might be overkill in this case.

Reply:
This is a great idea! Good future-proofing too. However, we'll deal with this later. Again, feel free to make a PR.